User/privileges.md

```diff
@@ -0,0 +1,82 @@
+# Privilege
+
+## Overview
+
+A **Privilege** is an authorization to perform a particular action in the system. The list of available privileges are defined by the core system and by add-on modules (for example, **Delete Patients** and **Manage Encounter Types**), but you need to configure which roles have which privileges while you are configuring your system.
+
+## Available operations.
+
+1. [List privilege](#list-privileges)
+2. [Create a privilege](#create-a-privilege)
+3. [Update a privilege](#update-a-privilege)
+4. [Delete a privilege](#delete-a-privilege)
+
+
+### List privilege
+
+* Fetch all non-retired privileges that match any specified parameters otherwise fetch all non-retired privileges. Returns a `200 OK` status with the privilege response. If the user not logged in to  perform this action, a `401 Unauthorized` status returned.
```

Comment:
[Privileges](https://docs.openmrs.org/datamodel/openmrs-refapp-data-model-1.9.html#%5Eprivilege) aren't voided or retired, just delete from the system when no longer needed. So, there aren't "retired" or "non-retired" privileges; rather, just privileges. Unlike many other data in OpenMRS, when a privilege is deleted, it is removed from the database (purged by default).